README.md

```diff
@@ -1,2 +1,29 @@
 # in-toto
 in-toto is a series of scripts to protect the supply chain integrity.
+
+# About in-toto
+A software supply chain is the series of steps performed when writing, testing, packaging, and distributing software. A typical software supply chain is composed of multiple steps "chained" together that transform (e.g., compilation) or verify the state (e.g., linting) of the project in order to drive it to a final product.
+
+In simple words, in-toto guarantees that the end-user (or owner) is able to verify that the entire development life cycle has been conducted as per the specified layout and that each of the functionaries (participants) have perfomed the specified tasks and there hasn't been any malicious changes in the files.
+
+So what in-toto basically does is that it asks for a project layout (JSON) stating the layout's expiry, functionary keys and steps involved. The project owner is tasked with layout specification. The layout also specifies the functionaries and the tasks they are supposed to perform.
```

Comment:
I'd personally drop the (JSON) part, considering that maybe we'll move on to another standard.